playbooks/roles/memcached/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: install memcached
+  apt: pkg=memcached state=present
```

Comment:
I personally don't like edX's Ansible conventions. They're mostly arbitrary and are not even consistently followed within the configuration repo (e.g. the capitalization scheme for variables). I suppose that it would be better to follow their conventions for things we push upstream, but I have no problem with the `module: foo=bar` syntax for our own code.

To be clear, using handlers in Ansible is _not_ considered an anti-pattern. Some people are against the use of handlers because they are not guaranteed to be executed if a playbook fails. Code in the configuration repo is so tangled and inter-dependent that it might be safer to not use handlers, but in general they're perfectly fine if you design your roles properly.